pymatgen/core/interface.py

```diff
@@ -2160,7 +2163,9 @@ def slab_from_csl(csl, surface, normal, trans_cry, max_search=20, quick_gen=Fals
                     ab_norm = ab_norm_temp
                     t_matrix[0] = ii[0]
                     t_matrix[1] = ii[1]
-                elif area_temp < area or (abs(area - area_temp) < 1.0e-8 and ab_norm_temp < ab_norm):
+
+                # DEBUG (@DanielYang59): ab_norm would certainly be unbound
+                elif area_temp < area or (abs(area - area_temp) < 1.0e-8 and ab_norm_temp < ab_norm):  # type: ignore  # temporary supress
```

Comment:
Seems a bug here, `ab_norm` would certainly be unbound:
https://github.com/materialsproject/pymatgen/blob/ad6eafee2474718813cba57aa8e1752f1181b639/pymatgen/core/interface.py#L2153-L2167